arte-decoder/src/System/Arte/Decode/Algorithm.hs

```diff
@@ -252,3 +254,33 @@ bound l h = unInf h . unZero l
 -- unused
 liftTC :: (a -> b) -> TrodeCollection a -> TrodeCollection b
 liftTC f tca = Map.map (Map.map f) tca
+
+
+
+------------------sending stuff-----------------------
+
+--remember to initialized myPort in ClusterlessOpts
+--initialize ipAddy (String) remember inet_addr :: String -> IO Host Address
```

Comment:
From http://linux.die.net/man/3/inet_addr

> Use of this function is problematic because -1 is a valid address (255.255.255.255). Avoid its use in favor of inet_aton(), inet_pton(3), or getaddrinfo(3) which provide a cleaner way to indicate error return.

Network.Socket exposes `getaddrinfo`